packages/dialtone-vue2/recipes/leftbar/contact_row/contact_row.vue

```diff
@@ -16,7 +16,7 @@
       <dt-avatar
         :full-name="name"
         :image-src="avatarSrc"
-        :image-alt="avatarAlt"
+        image-alt=""
```

Comment:
So we are saying that `image-alt=""` and `:image-alt="undefined"` are two different things? We set `image-alt=""` when the alt text is unnecessary because it's set set by another description?

Reply:
Yes, in fact it is.

If you set alt="undefined" or don't set it, it will raise an a11y warning.

If set to empty string, and there's text around, it will not raise any warning.

Reply:
https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

Reply:
Great, thanks for the article. I'm okay with this solution, but let's write in the comment for image-alt the difference between blank and undefined because you'd never know that if you didn't already read it.

Reply:
Comment updated